hibernate-core/src/test/java/org/hibernate/test/proxy/HibernateUnproxyTest.java

```diff
@@ -25,6 +27,41 @@
         return new Class<?>[]{Parent.class, Child.class};
```

Comment:
Add `SubChild` here as well, perhaps `SubChild` is just persisted as a `Child` currently..

Reply:
@JWGmeligMeyling Woooow you were right.. i've committed the SubChild.class.